src/types.rs

```diff
@@ -10,7 +10,7 @@
 
 use syntax::ast;
 use syntax::print::pprust;
-use syntax::codemap::{self, Span, BytePos, CodeMap};
+use syntax::codemap::{self, Span, BytePos, CodeMap, mk_sp};
```

Comment:
Thanks @marcusklaas , I used this for debugging but forgot to remove. PR updated.